local-dev/bin/render

```diff
@@ -2,26 +2,96 @@
 
 set -eo pipefail
 
-REPO_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
+PROJECT_ID="broad-dsde-dev"
 
-IMAGE_NAME="${IMAGE_NAME:-us-central1-docker.pkg.dev/dsp-artifact-registry/firecloud-develop-shim/firecloud-develop-shim}"
-IMAGE_TAG="${IMAGE_TAG:-latest}"
-IMAGE="${IMAGE_NAME}:${IMAGE_TAG}"
+# Helper function to fetch secrets
+fetch_secret() {
+  gcloud secrets versions access latest --project "$PROJECT_ID" --secret "$1"
+}
 
-VAULT_TOKEN=$( cat ~/.vault-token )
+##Pull mysql instance info for sqlproxy##
+RAWLS_MYSQL_INSTANCE_JSON=$(fetch_secret "rawls-mysql-instance")
 
-docker run --pull always \
-  -e VAULT_TOKEN="${VAULT_TOKEN}" \
-  -v "${REPO_ROOT}:/code" \
-  --rm -it "${IMAGE}" \
-  --secrets /code/local-dev/secrets.yaml \
-  --templates /code/local-dev/templates \
-  --output-dir /code/config
+# Extract the name and zone values using jq
+CLOUDSQL_ZONE=$(echo "$RAWLS_MYSQL_INSTANCE_JSON" | jq -r '.zone')
+CLOUDSQL_INSTANCE=$(echo "$RAWLS_MYSQL_INSTANCE_JSON" | jq -r '.name')
 
-cat <<EOF
+# Define the template and output file paths
+MYSQL_TEMPLATE_FILE="local-dev/templates/sqlproxy.env"
+MYSQL_OUTPUT_FILE="config/sqlproxy.env"
 
```

Comment:
add a `mkdir -p config` here, to ensure the `config` directory exists before we try to write to it?